smartcontractkit/external-adapters-js
# Feature/ Output testing

## Closes DF-189

## Description

Adds output testing functionality as part of soak testing. The output testing asserts output against specified values and helps to test the real adapter output. Also, output examples, posted along with the soak testing results, is a part of manual QA.

## Changes

Adds output testing script to k6 package.

See example of messages in https://github.com/smartcontractkit/external-adapters-js/pull/2300

## Steps to Test

See soak testing results:

Example of assertions failure: https://github.com/smartcontractkit/external-adapters-js/pull/2300#issuecomment-1321262585
Example of assertions pass: https://github.com/smartcontractkit/external-adapters-js/pull/2300#issuecomment-1321267600

## Quality Assurance
